docs/docs/configuration/skipping.md

```diff
@@ -34,12 +47,19 @@ Will skip any file named `bar` in the subdirectories of testdata.
 |     License      |     ✓     |
 
 By default, Trivy traverses directories and searches for all necessary files for scanning.
-You can skip directories that you don't maintain using the `--skip-dirs` flag.
+You can skip directories that you don't maintain using the `--skip-dirs` flag, or the equivalent YAML config option.
 
-```
+```bash
```

Comment:
Can we add a little subtitle *not bold or anything but just:

Using the `--skip-dirs` flag:

EXAMPLE

Using the Trivy YAML configuration:

EXAMPLE